src/main/drivers/compass/compass_hmc5883l.c

```diff
@@ -118,6 +120,7 @@
 #define HMC_POS_BIAS 1
 #define HMC_NEG_BIAS 2
 
+static busDevice_t *busdev;
```

Comment:
`busdev` shall be passed as argument instead of static

Reply:
Done.